src/modules/uavcan/sensors/baro.cpp

```diff
@@ -42,9 +42,11 @@ const char *const UavcanBarometerBridge::NAME = "baro";
 
 UavcanBarometerBridge::UavcanBarometerBridge(uavcan::INode& node) :
 UavcanCDevSensorBridgeBase("uavcan_baro", "/dev/uavcan/baro", BARO_BASE_DEVICE_PATH, ORB_ID(sensor_baro)),
-_sub_air_data(node),
+_sub_air_pressure_data(node),
```

Comment:
@LorenzMeier - this is the result of StaticAirData being broken up into  StaticTemperature and StaticPressure.

I am using last_temperature (below) to merge the call back data into the report. Is this acceptable?